sixtools/rixs_wrapper.py

```diff
@@ -0,0 +1,218 @@
+import numpy as np
+from scipy.interpolate import interp1d
+from pims import pipeline
+from rixs.process2d import apply_curvature, image_to_photon_events
+
+# Eventually we will create this information from the configuration
+# attributes in ophyd.
+process_dict_low_2theta = {'light_ROI': [slice(175, 1609), slice(1, 1751)],
+                           'curvature': np.array([0., 0., 0.]),
+                           'bins': None}
+
+process_dict_high_2theta = {'light_ROI': [slice(175, 1609), slice(1753, 3503)],
+                            'curvature': np.array([0., 0., 0.]),
+                            'bins': None}
+
+
+process_dicts = {'low_2theta': process_dict_low_2theta,
+                 'high_2theta': process_dict_high_2theta}
+
+
+@pipeline
+def image_to_spectrum(image, light_ROI=[slice(None, None, None),
```

Comment:
Hi @danielballan @stuwilkins 

Please can you glance over the core repo structure and the use of pims pipelines.

I want to know whether this is the correct bones to be developing from or not?

Reply:
This is the first time I've used a pims pipeline. 

To me, it makes sense to use pims on the basis that this is what DataBroker is returning. I assume this will continue to be a pims pipeline for the foreseeable future?

Reply:
Exactly what the DataBroker returns is controlled by runtime configuration (the "handler registry") which the user can easily modify. Since we are starting fresh, I might suggest not baking in `pims.pipeline` before we look at one or two alternatives. I volunteer to write some code to help.

How many images does this workflow normally handle at a time? How large are they?

Reply:
I just noticed your comment about wanting to merge this soon. I'm fine with punting this discussion to a separate issue; we just shouldn't consider the `image_to_spectrum` API to be set in stone yet.

Reply:
I didn't know that databroker could be reconfigured in this way. I looked at CSXtools for inspiration, which I assumed was done in an approved way.

Please do go ahead and write something. The full read-out of the images are ~1500*7000 pixels (half real pixels and half overscan). A scan would typically contain 10s to 100s of images.